var/spack/repos/builtin/packages/linux-external-modules/package.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from shutil import copy2
+
+
+class LinuxExternalModules(MakefilePackage):
+    """The Linux kernel provides services to start and stop programs,
+    handles the file system and other common "low-level" tasks that most
+    programs share, and schedules access to avoid conflicts when programs
+    try to access the same resource or device simultaneously. The kernel
+    has a modular design such that modules can be integrated as software
+    components. In this package, Linux has been configured to build out-of-tree
+    kernel modules."""
```

Comment:
(minor) It would be useful (but not necessary) to include a link to instructions on how to use these modules once Spack installs them (i.e. assuming you have `spack location -i linux-external-modules`, how you would configure linux to use them).

Reply:
Added.